Meta/lint-ports.py

```diff
@@ -346,11 +347,12 @@ def run():
 
     from_table_set = set(from_table.keys())
     ports_set = set(ports.keys())
+    ordered_list = sorted(from_table.keys(), key=str.lower)
```

Comment:
`ordered_list` is not really a descriptive name. `table_keys_sorted`?

Reply:
hmm… now i think that it should be something like `expected_ports_list` or something; i think i do not have a better name here—iʼll just restore previous version.